ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -47,6 +48,43 @@ const triggerSet: TriggerSet<Data> = {
         text: 'Split Tankbusters',
       },
     },
+    {
+      // First breaks north bridge for upcoming South Knockback Spreads
+      // Second breaks remaining bridges, Separate Healer Groups
+      // Third breaks all bridges, Bait on Empty Platform
+      id: 'P7S Roots of Attis 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.rootsCounter === undefined,
+      delaySeconds: 0.1, // Slight delay to prevent duplicate callout
+      run: (data) => data.rootsCounter = true,
+    },
+    {
+      id: 'P7S Roots of Attis 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.rootsCounter,
+      delaySeconds: 0.1, // Slight delay to prevent duplicate callout
+      infoText: (_data, _matches, output) => output.separateHealerGroups!(),
+      run: (data) => data.rootsCounter = false,
+      outputStrings: {
+        separateHealerGroups: {
+          en: 'Separate Healer Groups',
+        },
+      },
+    },
+    {
+      id: 'P7S Roots of Attis 3',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.rootsCounter === false,
+      infoText: (_data, _matches, output) => output.baitSoon!(),
+      outputStrings: {
+        baitSoon: {
+          en: 'Bait on Empty Platform Soon',
```

Comment:
Is it possible to have this call out which platform to go to?

Reply:
I'm not sure. Visually it is because you see which platforms have an egg spawn on them. I have not checked which npcIds spawn on which platforms, when that is logged, and if OverlayPlugin is needed and can obtain earlier data.

Reply:
I looked into it some more, there are Forbidden Fruit npcs that I am guessing are used, they map 1:1 with what I've seen for the Immature npcs.

11377 => Immature Minotaur (11380)
11376 => Immature Stymphalide (11379)
11375 => Immature Io (11378)

Reply:
Checking on spell cast for '7811', the forbidden fruits haven't moved. FFlogs seems to catch them as Agdistis. I might try adding some delay as I do see later '7811' casts have updated positions.

Reply:
A delay of 0.5s is enough to grab the data. It's probably available slightly sooner, but for latency reasons I think 0.5s has worked well.

Reply:
It seems either the delay is not consistent between phases, or the Forbidden Fruit combatant data is not what we see visually. I tried looking at Forbidden Fruit 7 here and the position data I am getting is not accurate. While the positions do change, they are not where the birds spawn. I might try and check the Agdistis entities next.

Reply:
Something I did just notice, the eggs do not appear until after the 7811 spell completes, so maybe that too could have been the problem.

Reply:
I have it working now, just checking for delay. Egg data is correct by 1s after 7811 ability.